addons/event/static/src/client_action/event_registration_summary_dialog.scss

```diff
@@ -0,0 +1,9 @@
+.o_event_barcode_dialog_status{
+    justify-content: center;
+    margin-left: 40px;
+}
+@include media-breakpoint-down(md) {
+    .o_event_barcode_dialog_status{
+        margin-left: 0px;
+    }
+}
```

Comment:
Would this work? (also, some spaces missing between selector and brackets ^^)
```scss
.o_event_barcode_dialog_status {
    justify-content: center;
    margin-left: 40px;
    @include media-breakpoint-down(md) {
        margin-left: 0px;
    }
}
```

Also note that if ms-4 or ms-5 is ok instead of 40px, we could replace all this file by directly adding
`justify-content-center ms-0 ms-md-5` on the divs (to check) It would be better :)

We can at least remove `justify-content: center;` by adding `justify-content-center` directly on the divs (we prefer limiting the custom css code)